src/lib/fb.js

```diff
@@ -141,6 +192,7 @@ export async function saveUserUrl(url, auditedOn = null) {
 
 // Sign in the user
 export async function signIn() {
+  await firebaseReady;
```

Comment:
I think this is a bit of a drawback for me. Over time it feels like we might need to sprinkle these ready checks or forgetting to do so might lead to weird bugs.

Reply:
We could always load app/auth (even as a module import side-effect from the actual CDN so it's happening in the same 'place' as the lazy-loading) but optionally load Firestore via the lazy-loading approach.

That way at least we'd reduce the complexity.

Reply:
yeah that might be good. I'm also curious to hear if David has any thoughts. I'm hoping there's a magical, well trod path that he can point us at 🤞